ansible_collections/arista/avd/docs/release-notes/5.x.x.md

```diff
@@ -176,6 +176,12 @@ Starting AVD 5.0.0, by default `redistribute igmp` will only get configured when
 
 See the [Porting guide for AVD 5.x.x](../porting-guides/5.x.x.md#removed-redistribute-igmp-from-bgp-vlan-config-belong-to-a-vrf-with-evpn-multicast)
 
+#### vrf_id will be used instead of vrf_vni to get the ip address of evpn underlay multicast group
+
+Starting AVD 5.0.0, `vrf_vni` will be replaced with `vrf_id` in the algorithm to to set the offset for ip address of evpn underlay multicast group when evpn_l3_multicast is enabled.
```

Comment:
```suggestion
Starting AVD 5.0.0, `vrf_id` is used instead of `vrf_vni` in the algorithm to set the offset for IP address of EVPN underlay multicast group when `evpn_l3_multicast` is enabled.
```